clients/js/.eslintrc.cjs

```diff
@@ -20,5 +20,6 @@ module.exports = {
   rules: {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-non-null-assertion': 'off',
+    eqeqeq: 1,
```

Comment:
Better to follow convention, I don't see 1 listed as one of the options: https://eslint.org/docs/latest/rules/eqeqeq#options

Reply:
I got the following error in the CLI initially:

```
Error: .eslintrc.cjs:
	Configuration for rule "eqeqeq" is invalid:
	Severity should be one of the following: 0 = off, 1 = warn, 2 = error.
```

So I changed it to `1` to warn but not error out.